### ISSUE TITLE
Adding to Inventry fails - x is invalid location

### ISSUE THREAD

**Issue by jdhirst** [EvEmu-Project/evemu_Crucible#161]
When trying to add Liquid Ozone to Jump Bridge resources, the following messages are displayed:
```
11:35:25 [InvMsg] IB::List() called by Kym with ownerID 90000000 for Cormorant(140000177:Cargo Hold) - origFlag: Cargo Hold
11:35:32 [Service] config::GetDynamicCelestials()
11:35:46 [Service] config::GetCelestialStatistic()
11:36:17 M     SystemManager: UnloadSystem() called for P3EN-E(30000250).
11:36:54 [InvMsg] IB::List() called by Kym with ownerID 98000000 for Jump Bridge(140000940:Hangar) - origFlag: Hangar
11:36:54 [InvMsg] Calling InventoryBound::GetItem() for Jump Bridge(140000940)
11:36:56 [InvMsg] Calling InventoryBound::GetItem() for Cormorant(140000177)
11:37:04 [Service] LSC::SendMessage()
11:37:09 [InvMsg] IB::Handle_Add() - moving 10000 Liquid Ozone(140001061) from (140000177:Cargo Hold) to me(Cormorant:140000177:Cargo Hold).
11:37:11 [InvMsg] IB::Handle_Add() - moving 10000 Liquid Ozone(140001062) from (140000177:Cargo Hold) to me(Jump Bridge:140000940:Hangar).
11:37:11 [ItemError] II::Donate() - 140000940 is invalid location
```

Yesterday when testing, the item was added but did not display in the client. However, during today's testing, the items are not being added at all due to the invalid location error above.

**Comment by jdhirst** [comment]
UPDATE: So it seems this is not limited to jump bridges, but also IHubs. Has something changed in the Inventory code recently that could have broken this? It was working yesterday.

**Comment by Almamu** [comment]
Everything that is not an station, solar system or region is not a valid location for an item looking at the code. Seems to happen since 16716e8, with the change to IsValidLocation

**Comment by zhyrohaad** [comment]
while updating the "IsValidLocationID" macro, i left out the check for "playerItemIDs".
while this method/macro only checks for the validity of a range, it is good enough for now.

**Comment by jdhirst** [comment]
This is now confirmed fixed as of my testing on 850d2105e01330fddc9cb52338fb5baf84641d52. Closing
